app/Http/Controllers/Front/PageController.php

```diff
@@ -766,35 +787,42 @@ public function getPriceDescription(int $productid)
     {
         try {
             $product = Product::find($productid);
-            if ($product['add_to_contact'] == 1) {
+
```

Comment:
we can even try this here public function getPriceDescription(int $productid)
{
    try {
        $product = Product::find($productid);

        if (!$product || $product->add_to_contact == 1) {
            return '';
        }

        $plans = Plan::with('planPrice')->where('product', $productid)->get();

        $priceDescription = match (true) {
            // Check for yearly plans (365 or 366 days)
            count($plans->filter(fn ($plan) => $plan->days == 365 || $plan->days == 366)) > 0 => 'free', // Assuming 'Free' means free
            // Default case, use the first planPrice if available
            default => $plans->first()?->planPrice->first()?->no_of_agents
                ? 'per month for <strong>'.$plans->first()->planPrice->first()->no_of_agents.' agent</strong>'
                : 'per month'
                : '',
        };

        return $priceDescription;
    } catch (\Exception $ex) {
        app('log')->error($ex->getMessage());
        return redirect()->back()->with('fails', $ex->getMessage());
    }
}